etc/kayobe/pulp.yml

```
@@ -373,6 +373,18 @@ stackhpc_pulp_rpm_repos:
     base_path: "opensearch-dashboards/2.x/yum/"
     required: "{{ stackhpc_pulp_sync_for_local_container_build | bool and stackhpc_pulp_sync_el_9 | bool }}"
 
+  - name: DOCA Online Repo {{ stackhpc_pulp_doca_version }} - RHEL 9.{{ stackhpc_pulp_repo_rocky_9_minor_version }}
+    url: "{{ stackhpc_release_pulp_content_url }}/doca/{{ stackhpc_pulp_doca_version }}/rhel9.{{ stackhpc_pulp_repo_rocky_9_minor_version }}/x86_64/{{ stackhpc_pulp_repo_rhel9_doca_version }}"
+    distribution_name: "doca-{{ stackhpc_pulp_doca_version }}-rhel9.{{ stackhpc_pulp_repo_rocky_9_minor_version }}-"
+    base_path: "doca/{{ stackhpc_pulp_doca_version }}/rhel9.{{ stackhpc_pulp_repo_rocky_9_minor_version }}/x86_64/"
+    required: "{{ stackhpc_pulp_sync_el_9 }}"
```

Should this be something like:
```suggestion
    required: "{{ dnf_enable_doca | bool and stackhpc_pulp_sync_el_9 }}"
```

---

It might be cleaner to just change the module sync variable to control the entire DOCA sync process.